src/services/carSaleTracking/buyerFeedback.ts

```diff
@@ -0,0 +1,20 @@
+import { postData, ApiCallOptions } from "../../base"
+import { WithValidationError } from "../../types/withValidationError"
+
+export const addPurchaseConfirmation = async ({
```

Comment:
To me the naming is a bit confusing as we add a feedback that either confirms or denies a purchase, might be jsut a personal view 🙃
It aligned with backed so it's fine...

Reply:
https://car-sale-tracking-service.preprod.carforyou.ch/swagger-ui/index.html#/Buyer%20Feedback/addPurchaseConfirmationUsingPOST

the naming comes straight from the api call: 
```
/api/car-sale/buyer-feedback/key/{key}/add-purchase-confirmation
```